### PR TITLE
Fix hostcheck

### PR DESCRIPTION
**Pull request by acomodi**
Based on https://github.com/SymbiFlow/prjxray/pull/1632 with the addition of https://github.com/SymbiFlow/prjxray/pull/1632#issuecomment-805061272

### CODE REVIEW

File: .github/kokoro/steps/hostcheck.sh [hunk 1]
```diff
@@ -11,21 +11,76 @@ set -e
...
+       echo ""
+       echo "systemctl | grep dev"
+       echo "----------------------------------------"
+       systemctl | grep dev
+
+       # Fail if we have waited to long.
+       if [[ $STORAGE_CHECK_ATTEMPTS -gt 5 ]]; then
+               exit $MOUNT_RET
+       else
+               STORAGE_CHECK_ATTEMPTS = $(( $STORAGE_CHECK_ATTEMPTS + 1 ))
```
Comment (dnltz) [inline .github/kokoro/steps/hostcheck.sh:78]: Without the spaces? https://source.cloud.google.com/results/invocations/094ac6d6-045b-4ad3-a3cb-28420642d9cb/targets/foss-fpga-tools%2Fprjxray%2Fpresubmit%2Fdatabase%2Fartix7/log
```suggestion
               STORAGE_CHECK_ATTEMPTS=$(( $STORAGE_CHECK_ATTEMPTS + 1 ))
```

**Comment by acomodi** [comment]
The hostcheck script keeps failing even after an increased amounts of retries, I am not sure how this can be fixed

**Comment by dnltz** [comment]
@acomodi - Can you please add "mount" to the debug output? The partition exists but seems like it's not mounted...

**Comment by acomodi** [comment]
@mithro @litghost For reference

Failing job:
- [zynq7](https://source.cloud.google.com/results/invocations/c89ac8e5-d50a-4d1b-be7c-18fb87bf2050/targets/foss-fpga-tools%2Fprjxray%2Fpresubmit%2Fdatabase%2Fzynq7/log)

Successfull job:
- [artix7](https://source.cloud.google.com/results/invocations/ec07e98b-f7f4-41d1-8cd1-c16beb4a2823/targets/foss-fpga-tools%2Fprjxray%2Fpresubmit%2Fdatabase%2Fartix7/log)
- [kintex7](https://source.cloud.google.com/results/invocations/2763e767-7839-4100-be72-b2523c6a59dd/targets/foss-fpga-tools%2Fprjxray%2Fpresubmit%2Fdatabase%2Fkintex7/log)

**Comment by litghost** [comment]
> Artix7 build seems to be gone in an infinite loop

Seems ok?  Just running.

**Comment by acomodi** [comment]
> Seems ok? Just running.

Ok, it was strange as it got restarted at a certain point, probably the first job failed or something.

**Comment by mithro** [comment]
I /think/ systemd seems to think that the device has started correctly mounted...
```
  sys-devices-pci0000:00-0000:00:03.0-virtio0-host0-target0:0:2-0:0:2:0-block-sdb-sdb1.device loaded active plugged   PersistentDisk tmpfs
```
However, I'm not seeing a `.mount` for `tmpfs`?

**Pull request merged**